server/agoraApi.yaml

```diff
@@ -180,7 +180,490 @@ paths:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Topic'
+        headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '400':
+          $ref: '#/components/responses/BadRequest' 
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/ForbiddenError'
+  /topics/{id}:
+    get:
+      tags:
+        - Topics
+      summary: Retrieves existing topic by Id
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: id of the topic to retrieve 
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/Topic'
+        headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/ForbiddenError'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+    delete:
+      tags: 
+        - Topics
+      summary: Delete an existing topic providing it's Id
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: id of the topic to delete 
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/Topic'
+        headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/ForbiddenError'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+  /resources:
+    get:
+      tags:
+        - Resources
+      summary: Retrieves all available resources, provide limit and offset for pagnation, top limit 100.
```

Comment:
Same feedback as lines 183 and 346 regarding either editing documentation to communicate that default and not max is 100, or editing service layer to enforce max 100.